app/src/main/java/com/moonpig/mvisample/productpicker/ProductPickerActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.moonpig.mvisample.productpicker
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.moonpig.mvisample.R
+import com.moonpig.mvisample.productdetail.ProductDetailActivity
+import kotlinx.android.synthetic.main.activity_product_picker.error
+import kotlinx.android.synthetic.main.activity_product_picker.success
+
+class ProductPickerActivity : AppCompatActivity() {
```

Comment:
Can you add a note here about why this doesn't follow MVI? i.e. that it's just a dummy screen to load the MVI ones etc